### PR TITLE
ROC-3566 send claim details with bulk print request

### PR DESCRIPTION
**Pull request by rattur**
Send claim details with additional data to bulk print request

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-3566

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java [hunk 1]
```diff
@@ -68,4 +71,12 @@ public void notifyStaffForBulkPrintFailure(DocumentReadyToPrintEvent event) {
         );
         appInsights.trackEvent(BULK_PRINT_FAILED, event.getClaim().getReferenceNumber());
     }
+
+    private static Map<String, Object> wrapInMap(Claim claim) {
+        Map<String, Object> additionalData = new HashMap<>();
+        additionalData.put("caseReferenceNumber", claim.getReferenceNumber());
+        additionalData.put("letterType", claim.getLetterHolderId());
```
Comment (timja-kainos) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java]: whats the point in this?

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java [hunk 2]
```diff
@@ -32,4 +32,5 @@
     private LocalDateTime respondedAt;
     private CCDSettlement settlement;
     private LocalDateTime settlementReachedAt;
+    private Long internalId;
```
Comment (timja-kainos) [inline ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java]: what's this?

Comment (geek-soft) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java]: @rattur this is no longer needed

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -106,3 +106,5 @@ smoke-test-citizen:
   password: "${SMOKE_TEST_USER_PASSWORD:}"
 smoke-test-solicitor:
   password: "${SMOKE_TEST_USER_PASSWORD:}"
+
+send_letter_url: 'http://localhost:4400'
```
[the remark this answers is not in the document]
Comment (rattur) [reply src/main/resources/application.yml]: Ok

File: src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java (hunk 1 above)
Comment (geek-soft) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java]: @rattur we do not want to mark print requests with letter holder ID. It does not make sense.

We just wanted to pass the type of the letter despite the fact there is only one CMC letter bulk printed ATM. Pass fix value like `defendantPack` or `defendantLetter` or similar.

**Pull request merged**